docs/examples/Track_Stitcher_Example.py

```diff
@@ -0,0 +1,585 @@
+            if id1 != id2:
+                total += 1
+                if id1[0] == id2[0] and id1[1] == (id2[1] - 1):
+                    count += 1
+    return (count / total * 100)
+
+
+print("Tracklets stitched correctly: ", StitcherCorrectness(stitched_tracks), "%")
+
+# %%
+# SIAP Metrics
+# ^^^^^^^^^^^^
+# The following cell calculates and records a range of SIAP (Single Integrated Air Picture) metrics to assess the
+# accuracy of the stitcher. The value of 'association_threshold' should be adjusted to represent the acceptable
+# distance for association for the scenaio that is being considered. For example, associating with a threshold of 50
+# metres may be acceptable if tracking a large ship, but not so useful for tracking cell movement.
+#
+# SIAP Ambiguity: Number of tracks assigned to a true object. Important as a value not equal to 1 suggests that the
+# stitcher is not stitching whole tracks together, or stitching multiple tracks into one.
+#
+# SIAP Completeness: Fraction of true objects being tracked. Not a valuable metric for track stitching evaluation as we
+# are only tracking fractions of the true objects - metric value is scaled by the ratio of truthlets to disjoint
+# sections.
+#
+# SIAP Longest Track Segment: Duration of longest associated track segment per truth.
+#
+# SIAP Position Accuracy: Positional error of associated tracks to their respective truths.
+#
+# SIAP Rate of Track Number Change: Rate of number of track changes per truth. Important metric for assessing track
+# stitching. Any value above zero is showing that tracklets are being incorrectly stitched to tracklets from different
+# truth paths.
+#
+# SIAP Spuriousness: Fraction of tracks assigned to a true object.
+#
+# SIAP Velocity Accuracy: Velocity error of associated tracks to their respective truths.
+
+import matplotlib.pyplot as plt
+from stonesoup.measures import Euclidean
+from stonesoup.metricgenerator.tracktotruthmetrics import SIAPMetrics
+from stonesoup.dataassociator.tracktotrack import TrackToTruth
+from stonesoup.metricgenerator.manager import SimpleManager
+from stonesoup.metricgenerator.metrictables import SIAPTableGenerator
+
+siap_generator = SIAPMetrics(position_measure=Euclidean((0, 2)),
+                             velocity_measure=Euclidean((1, 3)))
+
+associator = TrackToTruth(association_threshold=30)
+
+metric_manager = SimpleManager([siap_generator],
+                               associator=associator)
+metric_manager.add_data(truths, set(all_tracks))
+
+plt.rcParams["figure.figsize"] = (10, 8)
+metrics = metric_manager.generate_metrics()
+
+siap_averages = {metrics.get(metric) for metric in metrics
+                 if metric.startswith("SIAP") and not metric.endswith(" at times")}
+siap_time_based = {metrics.get(metric) for metric in metrics if metric.endswith(' at times')}
+
+_ = SIAPTableGenerator(siap_averages).compute_metric()
```

Comment:
Would be good to compare before and after, but I can see issue is that stitched tracks aren't interpolated. Probably should add that as separate component that could be used.